templates/zerver/emails/email_base_messages.html

```diff
@@ -4,21 +4,6 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
         <title>Zulip</title>
     </head>
-    <span style="display:none !important;
-      visibility:hidden;
-      color:transparent;
-      mso-hide:all;
-      font-size:1px;
-      color:hsl(0, 0%, 100%);
-      line-height:1px;
-      max-height:0px;
-      height:0px;
-      max-width:0px;
-      width:0px;
-      opacity:0;
-      overflow:hidden;">
-        {% block preheader %}{% endblock %}
-    </span>
```

Comment:
Won't this remove the preheader from the digest emails too?

Reply:
hmmm. We are not using the `preheader` block in digest.html email right?

Reply:
Yes, but we likely will want to in the future.  So it might make sense to not remove this (or make this block properly conditional on whether a preheader is being used).